test/src/unit-capi-query.cc

```diff
@@ -31,6 +31,7 @@
  * Tests for the C API tiledb_query_t spec.
  */
 
+#include <tiledb/sm/c_api/tiledb_struct_def.h>
```

Comment:
Is this needed?

Reply:
Nope leftover from the last commit. I will remove it